notebooks/community/model_garden/model_garden_pytorch_peft.ipynb

```diff
@@ -0,0 +1,947 @@
+{
```

Comment:
<p>use:</p><p>import sys</p><p>if 'google.colab' in sys.modules:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done</p>